python/lsst/ts/externalscripts/auxtel/make_latiss_calibrations.py

```diff
@@ -86,6 +86,17 @@ def get_schema(cls):
         description: Configuration for making a LATISS calibrations SAL Script.
         type: object
         properties:
+            detectors:
+                description: Detector IDs.
+                type: array
+                items:
+                  - type: number
+                minContains: 0
+                maxContains: 0
+                minItems: 0
+                maxItems: 1
```

Comment:
uhm.. do you really need this to be an array if it is only one item? Do you really need this at all for LATISS?